simpleflow/constants.py

```diff
@@ -1,3 +1,8 @@
 import os
 
 SIMPLEFLOW_ENV = os.getenv("SIMPLEFLOW_ENV", "production")
+
+# Constants useful for defining SWF timeouts
+MINUTE = 60
+HOUR = 60 * MINUTE
+DAY = 24 * HOUR
```

Comment:
:fearful: Oops! right, I should have put them in simpleflow/settings/, for dynamic import...

What's the UUID for Earth in this era? I've got a file to name :slightly_smiling_face: